app/apps/cases/tests/tests_api.py

```diff
@@ -0,0 +1,193 @@
+from apps.cases.models import Case, CaseReason, CaseTeam
+from django.urls import reverse
+from rest_framework import status
+from rest_framework.test import APITestCase
+
+from app.utils.unittest_helpers import (
+    get_authenticated_client,
+    get_test_user,
+    get_unauthenticated_client,
+)
+
+
+class CaseTeamApiTest(APITestCase):
+    def test_unauthenticated_get(self):
+        url = reverse("teams-list")
+        client = get_unauthenticated_client()
+        response = client.get(url)
+        self.assertEqual(response.status_code, status.HTTP_401_UNAUTHORIZED)
+
+    def test_authenticated_get(self):
+        url = reverse("teams-list")
+        client = get_authenticated_client()
+        response = client.get(url)
+        self.assertEqual(response.status_code, status.HTTP_200_OK)
+
+    def test_authenticated_get_empty(self):
+        url = reverse("teams-list")
+        client = get_authenticated_client()
+
+        response = client.get(url)
+        data = response.json()
+
+        self.assertEquals(data["results"], [])
+
+    def test_authenticated_get_filled(self):
+        CaseTeam.objects.create(name="Foo Case Team A")
+        CaseTeam.objects.create(name="Foo Case Team B")
```

Comment:
Zou model bakery gebruiken ipv create. Lange termijn als we andere modellen aan CaseTeam hangen of iets anders heeft invloed daarop, hoef je niet zoveel de testen te fixen

Reply:
Done! Heb de object creatie met model bakery vervangen in de tests